.licensure.yml

```diff
@@ -0,0 +1,57 @@
+change_in_place: true
+excludes:
+  - Cargo.toml
+  - Cargo.lock
+  - LICENSE
+  - pgx-utils/assets/ansi.tmTheme
+  - ".envrc"
+  - ".gitignore"
+  - "flake\\..*"
+  - "logo.*"
+  - ".github/**/*"
+  - ".cargo/config"
+  - "cargo-pgx/src/templates/*"
+  - ".*\\.control"
+  - ".*\\.md"
+  - ".*\\.nix"
+  - ".*\\.yml"
+licenses:
+  - files: any
+    ident: PostgreSQL
+    authors:
+      - name: ZomboDB, LLC and Technology Concepts & Design, Inc.
+        email: support@tcdi.com
+    template: |
```

Comment:
The PostgreSQL license does not require copyright headers in the original or derivative works. This could be removed.